src/components/DashboardWidget/DashboardWidget.vue

```diff
@@ -0,0 +1,385 @@
+			{{ item.title }}
+		</template>
+	</DashboardWidget>
+</template>
+
+<script>
+const myItems = [
+	{
+		title: 'first',
+		content: 'blabla',
+	},
+	{
+		title: 'second',
+		content: 'fuzzfuzz',
+	},
+]
+export default {
+	name: 'MyDashboardWidget',
+	props: [],
+	data() {
+		return {
+			items: myItems
+		}
+	},
+}
+</script>
+```
+
+### Complete example using DashboardWidgetItem
+
+```vue
+<template>
+	<DashboardWidget :items="items"
+		:show-more-url="'https://nextcloud.com'"
+		:item-menu="itemMenu"
+		@hide="onHide"
+		@markDone="onMarkDone">
+
+		<template #empty-content>
+			Nothing to display
+		</template>
+	</DashboardWidget>
+</template>
+
+<script>
+const myItems = [
+	{
+		id: '1',
+		targetUrl: 'https://target.org',
+		avatarUrl: 'https://avatar.url/img.png',
+		avatarUsername: 'Robert',
+		avatarIsNoUser: true,
+		overlayIconUrl: '/svg/core/actions/sound?color=000',
+		mainText: 'First item text',
+		subText: 'First item subtext',
+	},
+	{
+		id: '2',
+		targetUrl: 'https://other-target.org',
+		avatarUrl: 'https://other-avatar.url/img.png',
+		overlayIconUrl: '/svg/core/actions/add?color=000',
+		mainText: 'Second item text',
+		subText: 'Second item subtext',
+	},
+]
+const myItemMenu = {
+	// triggers an event named "markDone" when clicked
+	'markDone': {
+		text: 'Mark as done',
+		icon: 'icon-checkmark',
+	},
+	// triggers an event named "hide" when clicked
+	'hide': {
+		text: 'Hide',
+		icon: 'icon-toggle',
+	}
+}
+export default {
+	name: 'MyDashboardWidget',
+	props: [],
+	data() {
+		return {
+			items: myItems,
+			itemMenu: myItemMenu,
+			loading: true,
+		}
+	},
+	methods: {
+		onMoreClick() {
+			console.log('more clicked')
+			const win = window.open('https://wherever.you.want', '_blank')
+			win.focus()
+		},
+		onHide(item) {
+			console.log('user wants to hide item ' + item.id)
+			// do what you want
+		},
+		onMarkDone(item) {
+			console.log('user wants to mark item ' + item.id + ' as done')
+			// do what you want
+		},
+	},
+}
+</script>
+```
+
+ </docs>
+
+<template>
+	<div>
+		<EmptyContent v-if="showItemsAndEmptyContent && halfEmptyContentString && items.length !== 0"
+			class="half-screen"
+			:icon="halfEmptyContentIcon">
+			<template #desc>
+				{{ halfEmptyContentString }}
+			</template>
+		</EmptyContent>
+		<ul>
+			<li v-for="item in displayedItems" :key="item.id">
+				<!-- @slot The default slot can be optionally overridden. It contains the template of one item. -->
+				<slot name="default" :item="item">
+					<DashboardWidgetItem :id="item.id"
+						:target-url="item.targetUrl"
+						:avatar-url="item.avatarUrl"
+						:avatar-username="item.avatarUsername"
+						:avatar-is-no-user="item.avatarIsNoUser"
+						:overlay-icon-url="item.overlayIconUrl"
+						:main-text="item.mainText"
+						:sub-text="item.subText"
+						:item-menu="itemMenu"
+						v-on="handlers" />
+				</slot>
+			</li>
+		</ul>
+		<div v-if="loading">
+			<div v-for="i in 7" :key="i" class="item-list__entry">
```

Comment:
Would be cool to separate the sections a bit more with html comments for clarity :)